assets/js/util/insufficient-permissions-error-description.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * getInsufficientPermissionsErrorDescription function.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Gets a description for an insufficient permissions error.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} error Original error message.
+ * @param {Object} module Module data.
+ * @return {string} Error description.
+ */
+export function getInsufficientPermissionsErrorDescription( error, module ) {
+	const {
+		slug = '',
+		name = '',
+		owner = '',
```

Comment:
This should probably use `null` as default to indicate it's expected to be an object.

Reply:
Yep, updated.